sdks/python/apache_beam/typehints/schemas.py

```diff
@@ -267,12 +269,24 @@ def typing_to_runner_api(self, type_: type) -> schema_pb2.FieldType:
           logical_type=schema_pb2.LogicalType(urn=PYTHON_ANY_URN),
           nullable=True)
     else:
-      # TODO(bhulette): Add support for logical types that require arguments
-      return schema_pb2.FieldType(
-          logical_type=schema_pb2.LogicalType(
-              urn=logical_type.urn(),
-              representation=self.typing_to_runner_api(
-                  logical_type.representation_type())))
+      if logical_type.argument_type() is None:
+        return schema_pb2.FieldType(
+            logical_type=schema_pb2.LogicalType(
+                urn=logical_type.urn(),
+                representation=self.typing_to_runner_api(
+                    logical_type.representation_type())))
+      else:
+        # TODO(bhulette,yathu): Complete support for logical types that require
+        # arguments.
```

Comment:
Please file an issue for this and link it here. Also note that the option_to_runner_api and option_from_runner_api has logic for working with FieldValues, we should re-use that when we take this on.

Reply:
entered #23373